chart/templates/workers/worker-deployment.yaml

```diff
@@ -98,6 +98,9 @@ spec:
           {{- toYaml $podAnnotations | nindent 8 }}
         {{- end }}
     spec:
+      {{- if and (semverCompare ">=2.8.0" .Values.airflowVersion) .Values.workers.kerberosSidecar.enabled }}
```

Comment:
For now, the `shareProcessNamespace` is kept under `kerberosSidecar` section to keep it tied to kerberos. It can be moved out to workers section in case it has another use case in the near future.